dev/ush/make_ee2_links.sh

```diff
@@ -0,0 +1,327 @@
+              link_file "${system_prefix}.t${cyc}z.sfci006.nc" "${system_prefix}.t${cyc}z.increment.sfc.i006.nc"
+          fi
+          if [[ -f "${system_prefix}.t${cyc}z.sfci009.nc" ]]; then
+              link_file "${system_prefix}.t${cyc}z.sfci009.nc" "${system_prefix}.t${cyc}z.increment.sfc.i009.nc"
+          fi
+          if [[ -f "${system_prefix}.t${cyc}z.sfcanl.nc" ]]; then
+              link_file "${system_prefix}.t${cyc}z.sfcanl.nc" "${system_prefix}.t${cyc}z.analysis.sfc.a006.nc"
+          fi
+          if [[ -f "${system_prefix}.t${cyc}z.sfcinc.nc" ]]; then
+              link_file "${system_prefix}.t${cyc}z.sfcinc.nc" "${system_prefix}.t${cyc}z.increment.sfc.i006.nc"
+          fi
+          if [[ -f "sfc_inc.tile1.nc" ]]; then
+              for tile in {1..6}; do
+                  link_file "sfc_inc.tile${tile}.nc" "increment.sfc.i006.tile${tile}.nc"
+              done
+          fi
+          if [[ -f "${system_prefix}.t${cyc}z.cubed_sphere_grid_atminc.tile1.nc" ]]; then
+              for tile in {1..6}; do
+                  link_file "${system_prefix}.t${cyc}z.cubed_sphere_grid_atminc.tile${tile}.nc" "${system_prefix}.t${cyc}z.jedi_increment.atm.i006.tile${tile}.nc"
+              done
+          fi
+      fi
+      cd "${cwd}"
+      if [[ -d "${cwd}/${dir}/${cyc}/analysis/ocean" ]]; then
+          cd "${cwd}/${dir}/${cyc}/analysis/ocean"
+          if [[ -f "${system_prefix}.t${cyc}z.ocninc.nc" ]]; then
+              link_file "${system_prefix}.t${cyc}z.ocninc.nc" "${system_prefix}.t${cyc}z.mom6_increment.i006.nc"
+          fi
+      fi
+      cd "${cwd}"
+      if [[ -d "${cwd}/${dir}/${cyc}/analysis/ice" ]]; then
+          cd "${cwd}/${dir}/${cyc}/analysis/ice"
+          for ice_file in *.cice_model_anl.res.nc; do
+              if [[ -f "${ice_file}" ]]; then
+                  # This gets the first two fields of the filename separated by dots
+                  prefix=$(echo "${ice_file}" | cut -d. -f1-2)
+                  link_file "${ice_file}" "${prefix}.analysis.cice_model.res.nc"
+              fi
+          done
+      fi
+      cd "${cwd}"
+        if [[ -d "${cwd}/${dir}/${cyc}/analysis/snow" ]]; then
+            cd "${cwd}/${dir}/${cyc}/analysis/snow"
+            for snow_file in *.sfc_data.tile1.nc; do
+                if [[ -f "${snow_file}" ]]; then
+                    prefix=$(echo "${snow_file}" | cut -d. -f1-3)
+                    # Keep the date fields of the prefix (the last two fields)
+                    # Lop off the "snowinc."
+                    prefix_new=$(echo "${prefix}" | cut -d. -f2-)
+                    for tile in {1..6}; do
+                        link_file "${prefix}.tile${tile}.nc" "${prefix_new}.snow_increment.sfc_data.tile${tile}.nc"
+                    done
+                fi
+            done
+        fi
+    done
+    cd "${cwd}"
+done
+
+for dir in "${enkfgdas_list[@]}" "${enkfgfs_list[@]}"; do
+    cd "${dir}"
+    cycle_list=($(ls -d -- ?? || true ))
```

Comment:
Need to add: 

```
   # Determine the system prefix
    system_prefix=""
    case "${dir}" in
        enkfgdas.*) system_prefix="enkfgdas" ;;
        enkfgfs.*) system_prefix="enkfgfs" ;;
        *) echo "Unknown directory prefix: ${dir}"; exit 1 ;;
    esac
```

Reply:
I just added it in along with the `enkfgcdas` RUN.